orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationError.java

```diff
@@ -0,0 +1,9 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.core.commons.ClientError;
+import com.sap.ai.sdk.orchestration.model.ErrorResponse;
+
+/** Orchestration error response. */
+// CHECKSTYLE:OFF
+public class OrchestrationError extends ErrorResponse implements ClientError {}
```

Comment:
(Major)

I have to check, but I think we avoid convenience classes from extending from lowlevel API. To avoid breaking changes, due to future code generations.